docker/docker-entrypoint.sh

```diff
@@ -50,3 +65,35 @@ fi
 if [ "${CONVERT_TO_CSV}" = true ]; then
   bash /app/csv-converter.sh /app/output/"$outputDir"
 fi
+
+if [ "$SEND_REPORT_TO_SERVER" = true ] && [ "$all_vars_for_upload_set" = true ]; then
```

Comment:
@alexanderkiel , @bastianschaffer this should be potentially reconsidered to be done in java
regardless of the decision - this call is currently missing all the auth options (basic, oauth, token) the program itself provides

Reply:
Yes in case `SEND_REPORT_TO_SERVER` is set, I would not create a file at all and instead send the report to the server in Java.